src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -109,24 +109,6 @@ public void TestSupportedDiagnosticsMessageDescription()
             }
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/26717")]
```

Comment:
📝 The replacement for this test (and the ones above) is `VerifyStandardProperties`:

https://github.com/dotnet/roslyn/blob/73e98581cc6034e93812dc37e7cde7aa93e11f47/src/Analyzers/CSharp/Tests/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs#L21-L23

That test works, but currently we don't have a way to enforce creating that test in test classes that use the new test framework.